__tests__/nc_news_db.test.js

```diff
@@ -52,3 +51,32 @@ describe("GET api/topics", () => {
 			});
 	});
 });
+
+describe("GET /api/articles/:articles_id", () => {
+	it("status: 200, should respond with the requested article", () => {
+		return request(app)
+			.get("/api/articles/1")
+			.expect(200)
+			.then(({ body }) => {
+				expect(body).toEqual({
+					article_id: expect.any(Number),
```

Comment:
They could use a flexible matcher eg **objectContaining** or **.toMatchObject** Using more flexible jest matchers keeps the tests flexible if the test data were to change or extend in properties in the future.